support-frontend/assets/components/nav/nav.tsx

```diff
@@ -15,12 +15,14 @@ interface NavProps {
 	countryGroupIds: CountryGroupId[];
 	selectedCountryGroup: CountryGroupId;
 	subPath: string;
+	countryIsAffectedByVATStatus: boolean;
```

Comment:
You might want to have this as 
```suggestion
	countryIsAffectedByVATStatus?: boolean;
```